### ISSUE TITLE
Fix "every section"

### ISSUE THREAD

**Issue by pokey** [cursorless-dev/cursorless#879]
"every section" should refer to ever section in the grandparent section, but seems to select the grandparent itself 🤔.  Might be worth waiting for #629

**Comment by bra1nDump** [comment]
Had a weird desire today to reverse the order of all my markdown sections in a document. The document is a log file that was in chronological order, but as it got longer I realized I want the latest day to be up top. I said take every section, also tried from file take every section.

Both resulted in a single section being selected.

I also noticed the section is marked as legacy, don't want to signup for this one yet, since I already have cheatsheet commitment. Just expressing interest in migrating

**Comment by pokey** [comment]
yeah it's actually the last legacy scope in markdown. shouldn't be too bad to migrate, as we're actually using a better tree-sitter-markdown now that has a better structure

**Comment by phillco** [comment]
nb: it was broken when it was a legacy scope, when we migrated it to new style scopes we omitted it, https://github.com/cursorless-dev/cursorless/pull/2770 fixes it for good
